src/scripts/cronjobs/emailBreachAlerts.tsx

```diff
@@ -319,9 +313,6 @@ export async function poll(
                   utmCampaignId={utmCampaignId}
                   subscriber={recipient}
                   dataSummary={dataSummary}
-                  enabledFeatureFlags={
-                    subPlatFeatureFlagEnabled ? ["SubPlat3"] : []
-                  }
```

Comment:
I decided to remove the whole `enabledFeatureFlags` parameter because [exactly what I was afraid of](https://github.com/mozilla/blurts-server/pull/5717#discussion_r2005773311) had happened - in `<BreachAlertEmail>` I had checked whether `enabledFeatureFlags` included `Moscary`, but it was never included.